cmd/pinniped-proxy/Cargo.lock

```diff
@@ -222,12 +222,6 @@ dependencies = [
  "winapi",
 ]
 
-[[package]]
-name = "doc-comment"
-version = "0.3.3"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "fea41bba32d969b513997752735605054bc0dfa92b4c56bf1189f2e174be7a10"
```

Comment:
Why is this PR including any changes to pinniped-proxy?

Reply:
No, no, it is due to a base branch being outdated. I've properly rebased and targeted it